snooty.toml

```
@@ -15,3 +15,13 @@ csharp-docs-versioned = "https://mongodb.github.io/mongo-csharp-driver/{+csharp-
 
 java-version = "4.8"
 java-api = "https://mongodb.github.io/mongo-java-driver/{+java-version+}"
+
+[[banners]]
+targets = ["csharp.txt"]
+
+variant = "tip"
+value = """ \
+    The new .Net/C# driver documentation is now live. To learn more about using \
```

Issue:
In this sentence structure, it is slightly ambiguous whether the driver is new or the documentation is new.

Suggestion:
Identify that it's documentation as well as if they happen to know about the existing documentation, that it's a new version of that.

```suggestion
    A new version of the .Net/C# driver documentation is now live. To learn more about using \
```